lib/commands/file-movement.js

```diff
@@ -166,11 +167,13 @@ async function pushFileToRealDevice (device, remotePath, base64Data) {
       dstPath = pathInContainer;
       log.info(`Parsed bundle identifier '${bundleId}' from '${remotePath}'. ` +
         `Will put the data into '${dstPath}'`);
-      if (isDocuments(containerType)) {
-        ifuseArgs = ['-u', device.udid, '--documents', bundleId, mntRoot];
-      } else {
-        ifuseArgs = ['-u', device.udid, '--container', bundleId, mntRoot];
-      }
+      ifuseArgs = [
+        '-u',
+        device.udid,
+        `${_.lowerCase(containerType) === IFUSE_CONTAINER_DOCUMENTS ? '--documents' : '--container'}`,
```

Comment:
You probably want https://lodash.com/docs/4.17.11#toLower

Reply:
template string is not needed: `_.toLower(containerType) === IFUSE_CONTAINER_DOCUMENTS ? '--documents' : '--container'` is fine

Reply:
ah, nice. toLower is better in this context. 👍 
(I took a look at the code in my curiosity and found it calls toLowerCase. interesting..)